platform/darwin/src/logging_apple.mm

```diff
@@ -0,0 +1,56 @@
+#include <mbgl/util/logging.hpp>
+#include <mbgl/util/enum.hpp>
+
+#import <Foundation/Foundation.h>
+#if __has_builtin(__builtin_os_log_format)
+#import <os/log.h>
+#endif
+
+namespace mbgl {
+    
+void Log::platformRecord(EventSeverity severity, const std::string &msg) {
```

Comment:
@tmpsantos can you review this function please

Reply:
(Generally the `os_log_` family of macros should be used at the call site, so this may be better as a core define somewhere..)

Reply:
We may want to modify this function it a follow-up PR.